report/assets/styles.css

```diff
@@ -875,46 +867,8 @@
   color: var(--color-fail-secondary);
 }
 
-/* Perf load opportunity */
-
-.lh-load-opportunity__cols {
-  display: flex;
-  align-items: flex-start;
-}
-
-.lh-load-opportunity__header .lh-load-opportunity__col {
-  color: var(--color-gray-600);
-  display: unset;
-  line-height: calc(2.3 * var(--report-font-size));
-}
-
-.lh-load-opportunity__col {
-  display: flex;
-}
-
-.lh-load-opportunity__col--one {
-  flex: 5;
-  align-items: center;
-  margin-right: 2px;
-}
-.lh-load-opportunity__col--two {
-  flex: 4;
-  text-align: right;
-}
-
-.lh-audit--load-opportunity .lh-audit__display-text {
-  text-align: right;
-  flex: 0 0 7.5ch;
-}
-
-
 /* Sparkline */
```

Comment:
We should be able to remove the sparkline stuff along with the opportunity components since they aren't used anymore:

https://github.com/GoogleChrome/lighthouse/blob/c4066039a743772f179794b805cc1e74b27dc988/report/assets/templates.html#L97-L130